moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -375,21 +379,13 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
       hidden: hiddenColumns["project_feature"],
       sorting: false,
       render: (entry) => {
-        // if there are no features, project_feature is [null]
-        if (!entry?.project_feature[0]) {
+        if (!entry?.project_feature) {
           return "-";
         } else {
-          const signalIds = [];
-          entry.project_feature.forEach((projectFeature) => {
-            const signal = projectFeature?.properties?.signal_id;
-            if (signal) {
-              signalIds.push({
-                signal_id: signal,
-                knack_id: projectFeature.properties.id,
-              });
-            }
-          });
-          return <RenderSignalLink signals={signalIds} />;
+          const signals = entry.project_feature.filter(
+            (signal) => signal.signal_id && signal.knack_id
+          );
```

Comment:
these two columns (`signal_id` and `knack_id`) are currently nullable in the DB. we should change that after we run the transpose geom script, right?

Reply:
I agree on that. A feature in the `feature_signals` table should always have those two columns populated.